### PR TITLE
Supplement the licenses of support etcd discovery

### PR DESCRIPTION
**Pull request by hddong**
### _Why are the changes needed?_

Supplement the licenses.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by hddong** [comment]
cc @pan3793

**Comment by hddong** [comment]
cc @pan3793

**Comment by hddong** [comment]
cc @pan3793

File: licenses-binary/LICENSE-MIT-codehaus.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (pan3793) [inline licenses-binary/LICENSE-MIT-codehaus.txt:1]: for which jar?

File: LICENSE-binary [hunk 2]
```diff
@@ -279,6 +302,9 @@ io.netty:netty-transport-native-epoll
 io.netty:netty-transport-native-kqueue
 io.netty:netty-transport-native-unix-common
```
Comment (pan3793) [inline LICENSE-binary:303]: I remember the kqueue and native-unix removed from tgz, could you please check that?

File: licenses-binary/LICENSE-MIT-codehaus.txt (hunk 1 above)
Comment (hddong) [reply licenses-binary/LICENSE-MIT-codehaus.txt:1]: animal-sniffer-annotations-1.21.jar
https://github.com/apache/incubator-kyuubi/blob/a54daf27c2798d99cc3fb9af7e4ac17b54d6a3e8/dev/dependencyList#L19

File: LICENSE-binary (hunk 2 above)
Comment (hddong) [reply LICENSE-binary:303]: > I remember the kqueue and native-unix removed from tgz, could you please check that?

Sure, had checked. `io.netty:netty-transport-native-kqueue` removed and `native-unix` still contains.
I will remove `kqueue`.

**Pull request closed without merging**